tests/vectors_sig.c

```diff
@@ -282,6 +364,8 @@ static int sig_ver_vector(const char *method_name,
 	OQS_SIG_free(sig);
 	return ret;
 
+	/* __attribute__ doesn't work on windows, so "use" sigLen to make compiler happy */
+	(void)sigLen;
```

Comment:
Is this because when `OQS_ENABLE_SIG_SLH_DSA` is turned off, `sigLen` is unused?
Regardless, please move it to the beginning of the function. It doesn't make sense to have code after `return`.